bond_test.go

```diff
@@ -51,10 +63,23 @@ type User struct {
 	Username  string `db:"username"`
 }
 
+func (u User) AfterCreate(tx bond.Session) {
```

Comment:
I think this is great.. in fact, I think we should only support the interface `AfterCreate(tx bond.Session)` .. and drop the empty one